### ISSUE TITLE
The &msc options show mouse movement events instead of scroll events

### ISSUE THREAD

**Issue by ng-nicholas** [nickcoutsos/keymap-editor#274]
I am trying to create custom behavior for a rotary encoder and I have noticed that the latest mouse emulation features have been incorporated into the editor. However, mouse movement events are still being listed under &msc bahavior, instead of the expected scroll options.

**Comment by nickcoutsos** [comment]
Yeah, this one's a pain in the ass because the scroll and move behaviors are functionally the same (both are instances of the `zmk,behavior-input-two-axis` behavior) and even the different binding parameters really just boil down to different default distances.

I have one overlay for behaviors with `zmk,behavior-input-two-axis` and another for `&msc` with the scroll-specific treatment, but that means _custom_ made behaviors intended for scrolling won't match. This is the only case where I'm still using this way to match so the error went unnoticed until now.
